kapost-bootstrapper.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_development_dependency "semantic" "~>1.6"
```

Comment:
Just missing a `,` between the gem name and version.

Reply:
Thanks! nice catch